spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/BatchDataReader.java

```diff
@@ -75,6 +86,7 @@ CloseableIterator<ColumnarBatch> open(FileScanTask task) {
     InputFile location = getInputFile(task);
     Preconditions.checkNotNull(location, "Could not find InputFile associated with FileScanTask");
     if (task.file().format() == FileFormat.PARQUET) {
+      LOG.debug("data file is parquet; calling deleteFilter to get SparkDeleteFilter ...");
```

Comment:
Capital? But maybe remove this one, not sure it's adding a lot of value since you could just add the line in the `deleteFilter` code

Reply:
Removed.